data_hub_api/kotahi_docmaps/v1/codecs/evaluation.py

```diff
@@ -67,13 +67,15 @@ def get_evaluation_and_type_list_from_email_body(
 ) -> Optional[Sequence[dict]]:
     if email_body:
         evalution_list = []
+
         evaluation_summary_text = extract_elife_assessments_from_email(email_body)
         if evaluation_summary_text:
             evaluation_summary_dict = {
                 'evaluation_type': DOCMAP_EVALUATION_TYPE_FOR_EVALUATION_SUMMARY,
                 'evaluation_text': evaluation_summary_text
             }
             evalution_list.append(evaluation_summary_dict)
+
         public_reviews = extract_elife_public_reviews_from_email(email_body)
```

Comment:
Maybe that variable could be renamed to reflect that it could also be a joint review?